src/main/java/com/littleBeasts/entities/Beast.java

```diff
@@ -12,39 +11,35 @@
 
 import java.awt.*;
 
-@CollisionInfo(collisionBoxWidth = 14, collisionBoxHeight = 8, collision = true)
-
+@CollisionInfo(collisionBoxWidth = 14, collisionBoxHeight = 30, collision = true)
 public class Beast extends Creature implements IUpdateable {
     private CeEntity littleBeast;
     private String monsterName;
     private Image portrait;
     private BeastStats beastStats;
     private int x, y;
     private boolean spwaned;
+    private boolean playerBeast;
+    private Spawnpoint e;
 
-    public Beast(Beasts beasts, int x, int y) {
+    public Beast(Beasts beasts, int x, int y, boolean playerBeast) {
```

Comment:
I would synchronise this with the CeEntity, where the var is called wild. Then we wouldn't need to invert the boolean. Minor change, not really necessary, but would give it better readability and beauty imo.

Reply:
I'll look into it, sounds like a very good idea.

Reply:
Hmm.. just thought about it. "wild" has more implications than playerBeast. 
wild differenciates between a beast that can be caught or not. 
playerBeast is only for the GUI. So it is a tag, if the Beast should be rendered in the BattleHud or not.